src/main/app/forms/models/individualDetails.ts

```diff
@@ -0,0 +1,46 @@
+import { PartyDetails } from './partyDetails'
+import DateOfBirth from 'app/forms/models/dateOfBirth'
+import { PartyType } from 'forms/models/partyType'
+
+export class IndividualDetails extends PartyDetails {
+
+  dateOfBirth?: DateOfBirth
+  constructor () {
+    super()
+    this.type = PartyType.INDIVIDUAL.value
+  }
+
+  static fromObject (input?: any): IndividualDetails {
+    if (input == null) {
+      return input
+    }
+    let deserialized = new IndividualDetails()
+    Object.assign(deserialized, PartyDetails.fromObject(input))
+    deserialized.type = PartyType.INDIVIDUAL.value
+    if (input.dateOfBirth) {
+      deserialized.dateOfBirth = DateOfBirth.fromObject(input.dateOfBirth)
+    }
+    return deserialized
+  }
+
+  deserialize (input?: any): IndividualDetails {
+    if (input) {
+      Object.assign(this, new PartyDetails().deserialize(input))
+      this.type = PartyType.INDIVIDUAL.value
+      if (input.dateOfBirth) {
+        this.dateOfBirth = DateOfBirth.fromObject(input.dateOfBirth)
+      }
+    }
+    return this
+  }
+
+  isCompleted (isClaimant?: boolean): boolean {
+    let dobComplete: boolean = true
```

Comment:
use a ternary statement here,  + why would you default to true then set to true again two lines down